apis/bases/_.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
```

Comment:
not sure why it was moved from 10.0 to 9.2

Reply:
what version of operator sdk do you have? we are using 1.26 now

Reply:
maybe you also just need to delete bin/controller-gen and download the latest

Reply:
That (removing `bin/controller-gen`) did the trick: somehow I had an older version but now it adds `v0.10.0` as expected. Thanks (sorry for the noise around this part).